cli/Cargo.toml

```diff
@@ -48,7 +48,7 @@ itertools = { workspace = true }
 octocrab = { version = "0.32.0", features = ["stream"] }
 once_cell = { workspace = true }
 open = "5.1.2"
-reqwest = { version = "0.11.22", default-features = false, features = ["json","rustls-tls"] }
+reqwest = { version = "0.11.22", default-features = false, features = ["json", "rustls-tls", "stream"] }
```

Comment:
needs a rebase.

Reply:
done, but im not sure what about this line makes you think that - i do indeed need to add the stream feature to reqwest